### PR TITLE
add evaluation results menu item in sidebar

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Evaluation Results - add new menu item for Evaluation Results #351](https://github.com/nerubia/kh360-react/issues/351)

BE PR: [update auth add user_type in response #327](https://github.com/nerubia/kh360-nodejs/pull/327)

### CODE REVIEW

File: src/components/ui/icon/icons.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { lazy } from "react"
 
 export const icons = {
   Ban: lazy(async () => await import("./icons/ban-icon")),
+  ListChecks: lazy(async () => await import("./icons/list-checks-icon")),
```
Comment (jatnerubia) [inline src/components/ui/icon/icons.ts:5]: fix order

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

File: src/components/ui/icon/icons.ts (hunk 1 above)
Comment (emmaacha) [reply src/components/ui/icon/icons.ts]: Done ^^

**Review by jatnerubia — APPROVED** [review]
Cool

**Review by Dev-Nino — APPROVED** [review]

**Pull request merged**